examples/audio/melgan_spectrogram_inversion.py

```diff
@@ -0,0 +1,548 @@
+"""
+Title: MelGAN Based Spectrogram Inversion using Feature Matching
+Author: [Darshan Deshpande](https://twitter.com/getdarshan)
+Date created: 02/09/2021
+Last modified: 02/09/2021
+Description: Inversion of audio from mel-spectrograms using the MelGAN architecture and feature matching.
+"""
+"""
+## Introduction
+
+Autoregressive vocoders have been ubiquitous for a majority of the speech processing history
+but they lack parallelism. [MelGAN](https://arxiv.org/pdf/1910.06711v3.pdf) is a
+non-autoregressive, fully convolutional vocoder architecture used for purposes ranging
+from spectral inversion and speech enhancement to speech synthesis when used as a decoder
+with models like Tacotron2 or FastSpeech.
+
+In this tutorial, we will have a look at the MelGAN architecture and how it can achieve
+fast spectral inversion, i.e. conversion of spectrograms to audio waves. The MelGAN
+implemented in this tutorial is similar to the original implementation with only the
+difference of method of padding for convolutions where we will use 'same' instead of
+reflect padding.
+"""
+
+"""
+## Imports
+"""
+
+"""shell
+!pip install -qqq tensorflow_addons
```

Comment:
No need for the !, it gets added automatically.

Reply:
removed